moveit_simple/CMakeLists.txt

```diff
@@ -18,7 +18,8 @@ find_package(catkin REQUIRED COMPONENTS
 )
 
 find_package(Boost REQUIRED)
-find_package(Eigen REQUIRED)
+find_package(Eigen3 REQUIRED)
+# find_package(Eigen REQUIRED) --> FindEigen.cmake Module has been depricated in Kinetic, please use FindEigen3.cmake Module [find_package(Eigen3)]
```

Comment:
Did you leave this comment in for some reason?

Reply:
Oh I changed this but did not test for a while.. wanted to make sure I remembered to check this in case something broke.. I'll delete the comment..